### PR TITLE
Updating master README.md for Install individual components

### PR DESCRIPTION
**Pull request by nparkstar**
# Pull Request Template for Kubeflow manifests Issues

  - After install as  "Install individual components", I can not connect to the central dashboard.
  - And this is related closed issue #2800, the message "Cannot get MLMD objects from Metadata store." appeared when I install with "Install individual components" method.
  - This update has to be applied to v1.9.0 also, so I will make a PR for v1.9.0

## ✅ Contributor checklist
  - Make sure you have tested with kustomize. See [Installation Prerequisites](https://github.com/kubeflow/manifests#prerequisites)
  - All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

---     
 
> You can join the CNCF Slack and access our meetings at the [Kubeflow Community](https://www.kubeflow.org/docs/about/community/) website. Our channel on the CNCF Slack is here [**#kubeflow-platform**](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,6 +187,7 @@ Install cert-manager:
 
 ```sh
 kustomize build common/cert-manager/cert-manager/base | kubectl apply -f -
+kustomize build common/cert-manager/kubeflow-issuer/base | kubectl apply -f -
```
Comment (juliusvonkohout) [inline README.md:190]: Please cross check this with example/kustomization.yaml

File: README.md [hunk 2]
```diff
@@ -280,6 +287,17 @@ Install kubeflow roles:
...
 
+#### Kubeflow Istio Resources
+
+Create the Kubeflow Gateway, `kubeflow-gateway` and ClusterRole, 
+`kubeflow-istio-admin`.
+
+Install kubeflow istio resources:
+
+```sh
+kustomize build common/istio-1-22/kubeflow-istio-resources/base | kubectl apply -f -
```
Comment (juliusvonkohout) [inline README.md:298]: Please cross check this with example/kustomization.yaml

**Comment by juliusvonkohout** [comment]
Thank you very much for the PR. @diegolovison please take a second look.

File: README.md (hunk 2 above)
Comment (juliusvonkohout) [reply README.md:298]: Looks good in https://github.com/kubeflow/manifests/blob/da0255f10d875040c2d845cd61b7938236c0dfaa/example/kustomization.yaml#L38

File: README.md (hunk 1 above)
Comment (juliusvonkohout) [reply README.md:190]: Looks good in https://github.com/kubeflow/manifests/blob/da0255f10d875040c2d845cd61b7938236c0dfaa/example/kustomization.yaml#L38

File: README.md (hunk 2 above)
Comment (juliusvonkohout) [reply README.md:298]: Looks also good https://github.com/kubeflow/manifests/blob/da0255f10d875040c2d845cd61b7938236c0dfaa/example/kustomization.yaml#L59

**Comment by juliusvonkohout** [comment]
You need to sign the DCO, then ping me again and I can merge

**Comment by nparkstar** [comment]
> Thank you very much for the PR. You need to sign with DCO, then ping me again and I can merge.

I did.
Thanks

**Pull request merged**